index.js

```diff
@@ -38,7 +38,12 @@ function setup(fetch) {
           // this will be retried
           const res = await fetch(url, opts);
           debug('status %d', res.status);
-          if (res.status >= 500 && res.status < 600) {
+          if ((res.status >= 500 && res.status < 600) || res.status === 429) {
+            // NOTE: doesn't support http-date format
+            const retryAfter = parseInt(res.headers.get('retry-after'), 10);
+            if (retryAfter) {
```

Comment:
Can we add a maximum value for `retryAfter`? (ie. not retry if `retryAfter` is 2 hours). Maybe ~20s?

Reply:
Added the `maxRetryAfter` option which defaults to `20`.